openedx/features/caliper_tracking/transformers/enrollment_transformers.py

```diff
@@ -0,0 +1,22 @@
+"""
+Transformers for all the enrollment events
+"""
+
+
+def edx_course_enrollment_deactivated(current_event, caliper_event):
+    """
+    Caliper specific log for edx course unenrollment
+    """
+    caliper_object = {
+        "course_id": caliper_event['course_id'],
```

Comment:
Please make sure your code is consistent. Either use single quotes or double. My recommendation is to use double quotes for stuff like docstrings and use single quotes everywhere else.